### PR TITLE
add lb options

### PR DESCRIPTION
**Pull request by bellatoris**
## Checklist
- [ ] 충분한 양의 자동화 테스트를 작성했는가?
  - 계단정복지도 서비스는 사이드 프로젝트로 진행되는 만큼 충분한 QA 없이 배포되는 경우가 많습니다. 따라서 자동화 테스트를 꼼꼼하게 작성하는 것이 서비스 품질을 유지하는 데 매우 중요합니다.

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #172** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/172" target="_blank">Graphite</a> 
    * **PR #173** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/173" target="_blank">Graphite</a> 
      * **PR #174** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/174" target="_blank">Graphite</a> 
        * **PR #175** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/175" target="_blank">Graphite</a> 
          * **PR #176** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/176" target="_blank">Graphite</a> 
            * **PR #177** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/177" target="_blank">Graphite</a> 
              * **PR #181** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181" target="_blank">Graphite</a> 
                * **PR #182** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/182" target="_blank">Graphite</a> 
                  * **PR #183** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183?utm_source=stack-comment).

File: infra/terraform/k3s/lightsail_lb.tf [hunk 1]
```diff
@@ -15,19 +15,44 @@ resource "aws_lightsail_lb_attachment" "k3s" {
...
+    "admin-api.staircrusher.club",
+    "admin.dev.staircrusher.club",
+    "admin.staircrusher.club",
+    "api.dev.staircrusher.club",
+    "api.staircrusher.club",
+    "redash.staircrusher.club",
+    "staircrusher.club",
+    "www.staircrusher.club",
+  ]
+}
```
Comment (Zeniuus) [inline infra/terraform/k3s/lightsail_lb.tf:38]: 오 이거 이제 되는군요 ㅋㅋ

**Review by Zeniuus — APPROVED** [review]

**Comment by bellatoris** [comment]
@bellatoris started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Comment by bellatoris** [comment]
@bellatoris started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/183).

**Pull request merged**